io.github.bkbilly.lnxlink.yaml

```
@@ -0,0 +1,221 @@
+        url: https://files.pythonhosted.org/packages/11/f1/0f1436d1bb7fae54f557a9965ab2b61163d57853c6f4e2a97236973acd28/jc-1.25.3-py3-none-any.whl
+        sha256: ea17a8578497f2da92f73924d9d403f4563ba59422fbceff7bb4a16cdf84a54f
+      - type: git
+        url: https://git@github.com/bkbilly/lnxlink.git
+        commit: 0e52431
+      - type: file
+        url: https://files.pythonhosted.org/packages/15/d8/dd071918c040f50fa1cf80da16423af51ff8ce4a0f2399b7bf8de45ac3d9/nose-1.3.7-py3-none-any.whl
+        sha256: 9ff7c6cc443f8c51994b34a667bbcf45afd6d945be7477b52e97516fd17c53ac
+      - type: file
+        url: https://files.pythonhosted.org/packages/c4/cb/00451c3cf31790287768bb12c6bec834f5d292eaf3022afc88e14b8afc94/paho_mqtt-2.1.0-py3-none-any.whl
+        sha256: 6db9ba9b34ed5bc6b6e3812718c7e06e2fd7444540df2455d2c51bd58808feee
+      - type: file
+        url: https://files.pythonhosted.org/packages/18/c7/8c6872f7372eb6a6b2e4708b88419fb46b857f7a2e1892966b851cc79fc9/psutil-6.0.0.tar.gz
+        sha256: 8faae4f310b6d969fa26ca0545338b21f73c6b15db7c4a8d934a5482faa818f2
+      - type: file
+        url: https://files.pythonhosted.org/packages/f9/9b/335f9764261e915ed497fcdeb11df5dfd6f7bf257d4a6a2a686d80da4d54/requests-2.32.3-py3-none-any.whl
+        sha256: 70761cfe03c773ceb22aa2f671b4757976145175cdfca038c02654d061d6dcc6
+      - type: file
+        url: https://files.pythonhosted.org/packages/73/67/8ece580cc363331d9a53055130f86b096bf16e38156e33b1d3014fffda6b/ruamel.yaml-0.18.6-py3-none-any.whl
+        sha256: 57b53ba33def16c4f3d807c0ccbc00f8a6081827e81ba2491691b76882d0c636
+      - type: file
+        url: https://files.pythonhosted.org/packages/46/ab/bab9eb1566cd16f060b54055dd39cf6a34bfa0240c53a7218c43e974295b/ruamel.yaml.clib-0.2.8.tar.gz
+        sha256: beb2e0404003de9a4cab9753a8805a8fe9320ee6673136ed7f04255fe60bb512
+      - type: file
+        url: https://files.pythonhosted.org/packages/ca/1c/89ffc63a9605b583d5df2be791a27bc1a42b7c32bab68d3c8f2f73a98cd4/urllib3-2.2.2-py3-none-any.whl
+        sha256: a448b2f64d686155468037e1ace9f2d2199776e17f0a46610480d311f73e3472
+      - type: file
+        url: https://files.pythonhosted.org/packages/94/db/fd0326e331726f07ff7f40675cd86aa804bfd2e5016c727fa761c934990e/xmltodict-0.13.0-py2.py3-none-any.whl
+        sha256: aa89e8fd76320154a40d19a0df04a4695fb9dc5ba977cbb68ab3e4eb225e7852
+  - name: xdotool
+    buildsystem: simple
+    build-commands:
+    - make -j4 && PREFIX=./build make install
+    - cp -r ./build/*  /app/
+    sources:
+    - type: archive
+      url: https://github.com/jordansissel/xdotool/releases/download/v3.20211022.1/xdotool-3.20211022.1.tar.gz
+      sha256: 96f0facfde6d78eacad35b91b0f46fecd0b35e474c03e00e30da3fdd345f9ada
+  - name: metadata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 io.github.bkbilly.lnxlink.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo
+      - install -Dm644 io.github.bkbilly.lnxlink.png -t $FLATPAK_DEST/share/icons/hicolor/scalable/apps
```

```suggestion
      - install -Dm644 io.github.bkbilly.lnxlink.png -t $FLATPAK_DEST/share/icons/hicolor/512x512/apps
```

Why is png going to scalable folder?